### PR TITLE
God Mode Fix

### PR DESCRIPTION
**Pull request by r0adra93**
set the attachentitytoentity to the proper settings according to fivem

**Describe Pull request**

working on semi-god mode issues with the suit.

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes] (Be honest)
- Does your code fit the style guidelines? [yes] (Be honest)
- Does your PR fit the contribution guidelines? [yes] (Be honest)

### CODE REVIEW

**Review by doublado — APPROVED** [review]

File: client/main.lua [hunk 1]
```diff
@@ -294,17 +294,19 @@ RegisterNetEvent('qb-diving:client:UseGear', function()
                         end
                         currentGear.tank = CreateObject(tankModel, 1.0, 1.0, 1.0, 1, 1, 0)
                         local bone1 = GetPedBoneIndex(ped, 24818)
-                        AttachEntityToEntity(currentGear.tank, ped, bone1, -0.25, -0.25, 0.0, 180.0, 90.0, 0.0, 1, 1, 0,
-                            0, 2, 1)
+                        AttachEntityToEntity(currentGear.tank, ped, bone1, -0.25, -0.25, 0.0, 180.0, 90.0, 0.0, true, true, false, false, 2, true)
+                        SetEntityCollision(currentGear.tank,false,true)
+                        SetEntityCollision(bone1,false,true)
```
Comment (zwkf) [inline client/main.lua:299]: Missing spaces between arguments

File: client/main.lua [hunk 2]
```diff
@@ -294,17 +294,19 @@ RegisterNetEvent('qb-diving:client:UseGear', function()
...
                         while not HasModelLoaded(maskModel) do
                             Wait(0)
                         end
                         currentGear.mask = CreateObject(maskModel, 1.0, 1.0, 1.0, 1, 1, 0)
                         local bone2 = GetPedBoneIndex(ped, 12844)
-                        AttachEntityToEntity(currentGear.mask, ped, bone2, 0.0, 0.0, 0.0, 180.0, 90.0, 0.0, 1, 1, 0, 0, 2
-                            , 1)
+                        AttachEntityToEntity(currentGear.mask, ped, bone2, 0.0, 0.0, 0.0, 180.0, 90.0, 0.0, true, true, false, false, 2, true)
+                        SetEntityCollision(currentGear.mask,false,true)
+                        SetEntityCollision(bone2,false,true)
```
Comment (zwkf) [inline client/main.lua:309]: Missing spaces between arguments

File: client/main.lua [hunk 3]
```diff
@@ -294,17 +294,19 @@ RegisterNetEvent('qb-diving:client:UseGear', function()
...
                         RequestModel(maskModel)
                         while not HasModelLoaded(maskModel) do
                             Wait(0)
                         end
                         currentGear.mask = CreateObject(maskModel, 1.0, 1.0, 1.0, 1, 1, 0)
                         local bone2 = GetPedBoneIndex(ped, 12844)
-                        AttachEntityToEntity(currentGear.mask, ped, bone2, 0.0, 0.0, 0.0, 180.0, 90.0, 0.0, 1, 1, 0, 0, 2
-                            , 1)
+                        AttachEntityToEntity(currentGear.mask, ped, bone2, 0.0, 0.0, 0.0, 180.0, 90.0, 0.0, true, true, false, false, 2, true)
+                        SetEntityCollision(currentGear.mask,false,true)
```
Comment (zwkf) [inline client/main.lua:308]: Missing spaces between arguments

File: client/main.lua [hunk 4]
```diff
@@ -294,17 +294,19 @@ RegisterNetEvent('qb-diving:client:UseGear', function()
                         end
                         currentGear.tank = CreateObject(tankModel, 1.0, 1.0, 1.0, 1, 1, 0)
                         local bone1 = GetPedBoneIndex(ped, 24818)
-                        AttachEntityToEntity(currentGear.tank, ped, bone1, -0.25, -0.25, 0.0, 180.0, 90.0, 0.0, 1, 1, 0,
-                            0, 2, 1)
+                        AttachEntityToEntity(currentGear.tank, ped, bone1, -0.25, -0.25, 0.0, 180.0, 90.0, 0.0, true, true, false, false, 2, true)
+                        SetEntityCollision(currentGear.tank,false,true)
```
Comment (zwkf) [inline client/main.lua:298]: Missing spaces between arguments

**Pull request closed without merging**